local ReplicatedStorage = game:GetService("ReplicatedStorage")
local Managers = script.Parent.Modules.Managers
local Services = script.Parent.Modules.Services
local SharedManagers = ReplicatedStorage.Modules.Shared.Managers

local RemotesManager = require(ReplicatedStorage.Modules.Shared.Managers.RemotesManager)
local PlayerDataManager = require(Managers.PlayerDataManager)
local PlayerStatsManager = require(SharedManagers.PlayerStatsManager)
local MorphManager = require(Managers.MorphManager)

local PlayerDataService = require(Services.PlayerDataService)
local MapService = require(Services.MapService)
local DevProductsService = require(Services.DevProductsService)
local GearsService = require(Services.GearsService)
local ItemsService = require(Services.ItemsService)
local GiftsService = require(Services.GiftsService)
local UpgradesService = require(Services.UpgradesService)
local DailyRewardsService = require(Services.DailyRewardsService)
local LeaderboardService = require(Services.LeaderboardService)
local GroupMorphService = require(Services.GroupMorphService)
local DiamondSlapService = require(Services.DiamondSlapService)
local SettingsService = require(Services.SettingsService)
local LeaderstatsService = require(Services.LeaderstatsService)

RemotesManager.Init()
PlayerDataManager.Init()
PlayerStatsManager.Init()
MorphManager.Init()
PlayerDataService.Init()
MapService.Init()
DevProductsService.Init()
GearsService.Init()
ItemsService.Init()
GiftsService.Init()
UpgradesService.Init()
DailyRewardsService.Init()
LeaderboardService.Init()
GroupMorphService.Init()
DiamondSlapService.Init()
SettingsService.Init()
LeaderstatsService.Init()

PlayerDataManager.Start()
PlayerStatsManager.Start()
MorphManager.Start()
PlayerDataService.Start()
MapService.Start()
DevProductsService.Start()
GearsService.Start()
ItemsService.Start()
GiftsService.Start()
UpgradesService.Start()
DailyRewardsService.Start()
LeaderboardService.Start()
GroupMorphService.Start()
DiamondSlapService.Start()
SettingsService.Start()
LeaderstatsService.Start()
