local ReplicatedStorage = game:GetService("ReplicatedStorage")

local Modules = ReplicatedStorage.Modules
local Shared = Modules.Shared
local Client = Modules.Client

local RemotesManager = require(Shared.Managers.RemotesManager)
local PlayerDataManager = require(Client.Managers.PlayerDataManager)
local GUIManager = require(Client.Managers.GUIManager)
local SoundManager = require(Client.Managers.SoundManager)

local CameraController = require(Client.Controllers.CameraController)
local MapController = require(Client.Controllers.MapController)
local TopSideController = require(Client.Controllers.TopSideController)
local LeftSideController = require(Client.Controllers.LeftSideController)
local RightSideController = require(Client.Controllers.RightSideController)
local GiftsController = require(Client.Controllers.GiftsController)
local UpgradesController = require(Client.Controllers.UpgradesController)
local DailyRewardsController = require(Client.Controllers.DailyRewardsController)
local DiamondSlapController = require(Client.Controllers.DiamondSlapController)
local TopbarController = require(Client.Controllers.TopbarController)
local SettingsController = require(Client.Controllers.SettingsController)
local MapAnimationController = require(Client.Controllers.MapAnimationController)
local MultipliersController = require(Client.Controllers.MultipliersController)

RemotesManager.Init()
PlayerDataManager.Init()
GUIManager.Init()
SoundManager.Init()
CameraController.Init()
MapController.Init()
TopSideController.Init()
LeftSideController.Init()
RightSideController.Init()
GiftsController.Init()
UpgradesController.Init()
DailyRewardsController.Init()
DiamondSlapController.Init()
TopbarController.Init()
SettingsController.Init()
MapAnimationController.Init()
MultipliersController.Init()

PlayerDataManager.Start()
GUIManager.Start()
SoundManager.Start()
CameraController.Start()
MapController.Start()
TopSideController.Start()
LeftSideController.Start()
RightSideController.Start()
GiftsController.Start()
UpgradesController.Start()
DailyRewardsController.Start()
DiamondSlapController.Start()
TopbarController.Start()
MapAnimationController.Start()
SettingsController.Start()
MultipliersController.Start()
